test/form/Form.spec.jsx

```diff
@@ -1,53 +1,117 @@
-import Bluebird from 'bluebird'
-import Immutable from 'immutable'
-import React from 'react'
-import TestUtils from 'react-testutils-additions'
-import sinon from 'sinon'
-import expect from 'unexpected'
-
-import ErrorMessage from '../../src/form/ErrorMessage'
-import Form from '../../src/form/Form'
-import {PropsSetter} from '../reactHelpers'
-import TestField from './TestField'
-
-function render (props) {
+import Bluebird from "bluebird";
+import Immutable from "immutable";
```

Comment:
Why are you adding semicolons all over the place?
Can't you turn of Prettier or

> tweak your prettier settings to match the existing code style

?

Reply:
Oops, I realised I updated only the `Form` itself but not the test